common/changes/@rushstack/heft/ianc-heft-config-loading_2020-09-12-01-44.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "Rename the \"outFolderPath\" and \"emitFolderPathForJest\" properties in typescript.json to \"outFolderName\" and \"emitFolderNameForJest\"",
```

```suggestion
      "comment": "(BREAKING CHANGE) Rename the \"outFolderPath\" and \"emitFolderPathForJest\" properties in typescript.json to \"outFolderName\" and \"emitFolderNameForJest\"",
```

Breaking changes need to be called out very clearly, so people aren't confused when they upgrade.